### DISCUSSION TITLE
O2 concentration and extinguishing

### DISCUSSION THREAD

**Discussion by HunterXHunter1997** [firemodels/fds#13378] [Q&A]
Hello team!
I have a nozzle that ejects Dodecane diesel into a room, during the fire time I reduced the room O2 volume fraction from 0.21 to 0.1 but the fire wasn't extinguished. (the fire has to be extinguished between 0.14-0.16 O2 level)
 
Here is the fire reaction that I defined:
&REAC ID='Diesel',
      FUEL='DODECANE',
      AUTO_IGNITION_TEMPERATURE=205.0,
      CO_YIELD=6.0E-3,
      SOOT_YIELD=0.015,
      HEAT_OF_COMBUSTION=4.8E4,
      RADIATIVE_FRACTION=0.35/
      
 What may cause this issue?
Thank you!

**Comment by drjfloyd** [comment]
Please submit a simple input file that demonstrates the problem you are seeing.
